### PR TITLE
Set version during release

### PR DESCRIPTION
**Pull request by jpreese**
I always get bit by this. Just doing some future-proofing with setting the version.

### CODE REVIEW

File: internal/commands/default.go [hunk 1]
```diff
@@ -7,13 +7,16 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// version is set during the build process.
+var version = ""
```
Comment (jalseth) [inline internal/commands/default.go:11]: nit: Here we're assigning an empty string rather than using `var version string`? 😛

Comment (jpreese) [reply internal/commands/default.go:11]: .. 🤦 . This is quality content.

**Comment by jpreese** [comment]
@jalseth I've pushed some changes that address the two issues you've outlined.

For the `release` target, typically yes. The input would be a tag that is pulled from CI and just set automatically. This was just the path of least resistance as I typically just run `make release` and drop the resulting binaries.

However I.. often.. forget to update the version. So now when `make release` is ran, it'll just flat out error without a version being set.

**Comment by jalseth** [comment]
Thanks for the clarification. LGTM!

**Pull request merged**